Userland/Applets/ClipboardHistory/main.cpp

```diff
@@ -43,7 +48,19 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     };
 
     auto delete_action = GUI::CommonActions::make_delete_action([&](const GUI::Action&) {
-        model->remove_item(table_view->selection().first().row());
+        if (table_view->selection().is_empty())
+            return;
+
+        auto index = table_view->selection().first();
+        model->remove_item(index.row());
+        if (index.row() == 0) {
+            if (model->is_empty()) {
```

Comment:
You can hoist this block out of the `if (index.row() == 0) {` check

Reply:
done :^)